scrapers/5Kteens.yml

```
@@ -0,0 +1,31 @@
+name: 5Kteens
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - 5kteens.com/episodes
+      - 5kporn.com/episodes
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Studio:
+        Name:
+          fixed: 5Kteens
+      Title: //p[@class="trailer-title"]
+      Details: //p[@class=""]
+      Performers:
+        Name:
+          selector: //h5[text()[contains(.,'Starring')]]
+          postProcess:
+            - replace:
+                - regex: Starring:\s([\w\s]+)
+                  with: $1
+      Date:
+        selector: //h5[text()[contains(.,'Published')]]
+        postProcess:
+            - replace:
+                - regex: Published\:\s(\d{4}-\d{2}-\d{2})
+                  with: $1
+
+driver:
+  useCDP: true
```

Good news, you don't need CDP for this scraper, all you need is to set a cookie :)
Example: https://github.com/stashapp/CommunityScrapers/blob/5b62f5ca33879e5acef06e3273d0c575804bb153/scrapers/KellyMadisonMedia.yml#L42-L48

For this scraper:
```suggestion
  cookies:
    - CookieURL: "https://www.5kteens.com/"
      Cookies:
        - Name: "nats"
          Domain: ".5kteens.com"
          ValueRandom: 23
          Path: "/"
    - CookieURL: "https://www.5kporn.com/"
      Cookies:
        - Name: "nats"
          Domain: ".5kporn.com"
          ValueRandom: 23
          Path: "/"
```